### ISSUE TITLE
Stack-traces showing source at line number where error occurred in bundle

### ISSUE THREAD

**Issue by matootie** [getsentry/sentry-webpack-plugin#232]
### TL;DR

I'm using Webpack with Sentry and Serverless (AWS Lambda). Error stack-traces are showing up as source but the line numbers are from where the error appears in the bundle, so it's way off.

### Description

@boriskurikhin and I have a Serverless project on AWS Lambda for which I am trying to set up Sentry. I went through the following documentation:

https://docs.sentry.io/platforms/node/guides/aws-lambda/
https://github.com/getsentry/sentry-webpack-plugin
https://docs.sentry.io/product/cli/configuration/

Everything *seems* to be working O.K, but then when I check the context for issues in the Sentry dashboard, I get very strange behaviour with the line numbers...

I'm really not sure where the problem is coming from. I'm kind of new to Webpack, Sentry *and* Serverless so... I'm out of ideas. If anyone has any ideas I could really use the help. Below I've added a bunch of context that I hope will help.

### Context

<details><summary>Screenshot of the stack-trace shown in the issue context. It's displaying the code from the source-map at the line where the error occurred <i>in the bundle</i>.</summary>
<p>
<img width="1392" alt="stacktrace" src="https://user-images.githubusercontent.com/31599272/95638593-285aa000-0a63-11eb-9e0e-09f054b1fa15.png">
</p>
</details>

<details><summary>Screenshot showing exactly where the error is coming from. <i>Line 30</i>.</summary>
<p>
<img width="954" alt="Screen Shot 2020-10-09 at 19 09 36" src="https://user-images.githubusercontent.com/31599272/95638679-7a9bc100-0a63-11eb-97a3-510444b8a9db.png">
</p>
</details>

<details><summary>Here is where I configure Sentry.</summary>
<p>

```js
const Sentry = require('@sentry/serverless');
const { RewriteFrames } = require('@sentry/integrations');

function withSentry(handler) {
  Sentry.init({
    dsn: process.env.SENTRY_DSN,
    integrations: [new RewriteFrames()]
  });

  return Sentry.AWSLambda.wrapHandler(handler);
}

module.exports = { withSentry };
```

</p>
</details>

<details><summary>Here is my <code>webpack.config.js</code>.</summary>
<p>

```js
const path = require('path');
const slsw = require('serverless-webpack');
const SentryCliPlugin = require('@sentry/webpack-plugin');

module.exports = {
  entry: slsw.lib.entries,
  target: 'node',
  devtool: 'source-map',
  mode: 'none',
  optimization: { minimize: false },
  performance: { hints: false },
  externals: { 'aws-sdk': 'aws-sdk' },
  module: {
    rules: [
      {
        test: /\.js$/,
        exclude: /node_modules/,
        use: [
          {
            loader: 'babel-loader',
          },
        ],
      },
    ],
  },
  plugins: [
    new SentryCliPlugin({
      include: './src/',
      ignore: ['node_modules', 'webpack.config.js', 'prettier.config.js']
    }),
  ],
  output: {
    libraryTarget: 'commonjs',
    path: path.join(__dirname, '.webpack'),
    filename: '[name].js',
  },
};
```

</p>
</details>

<details><summary>Here is a screenshot of the resulting artifacts uploaded to Sentry.</summary>
<p>

<img width="1436" alt="artifacts" src="https://user-images.githubusercontent.com/31599272/95638825-044b8e80-0a64-11eb-8926-e47560350b1b.png">

</p>
</details>

<details><summary>Here is a link to the Sentry issue.</summary>
<p>

https://sentry.io/share/issue/156c499f1e5a4018b1c32ad5634871b7/
https://sentry.io/organizations/klassroom/issues/1947597764/

</p>
</details>

**Comment by kamilogorek** [comment]
Everything looks correctly here, so it looks like the main issue is with producing valid artifacts in the first place.
Closing the issue as a part of large repository cleanup, due to it being inactive and/or outdated (Sentry issues link are already dead).
Please do not hesitate to ping me if it is still relevant, and I will happily reopen it.
Cheers!
